### ISSUE TITLE
Metrics from prometheus recording rules are dropped by wavefront

### ISSUE THREAD

**Issue by sandeepbhojwani** [wavefrontHQ/prometheus-storage-adapter#5]
Metrics generated by prometheus recording rules don't have `instance` label. Wavefront storage adapter expects this label and uses it to fill the value for `source`. Missing `instance` label in metrics results in an empty `source` generated by the storage adapter and metric with empty `source` is rejected by wavefront. 

It would be nice if the storage adapter can add a config that replaces empty or missing instance with a supplied value or an expression that picks up the value from some other labels.

The workaround i have used is to write a writeRelabelConfig in the remoteWrite that adds the instance label. 
```
      writeRelabelConfigs:
      - action: replace
        sourceLabels:
        - pod
        regex: (.*)
        replacement: '$1'
        targetLabel: instance
```

**Comment by vikramraman** [comment]
Thanks for raising the issue @sandeepbhojwani. We will take a look into it.

**Comment by vikramraman** [comment]
Addressed by #14.
